src/Navigation.css

```diff
@@ -0,0 +1,61 @@
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 0 38px;
+  padding: 15px 80px;
+  background-color: #fff;
+}
+
+.header {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  height: 45px;
+}
+
+.header__link {
+  font-family: sans-serif, Montserrat;
+  cursor: pointer;
+  font-weight: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  font-size: 13px;
+  height: 16px;
+  margin: 20px 40px;
+  opacity: 0.5;
+}
+
+.header__logo {
+  margin-left: 0;
+  height: 30px;
+  font-size: 26px;
+  letter-spacing: normal;
+  color: #0290ff;
+  font-weight: bold;
+  opacity: 1;
+}
+
+.header__book_link {
+  opacity: 1;
+}
+
+.user {
+  width: 35px;
+  height: 35px;
+  padding: 10px;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-self: center;
+}
+
+.user-logo {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: block;
+  width: 15px;
+  height: 15px;
+}
```

Comment:
- [ ] Styling should be EXACTLY the same as the one provided in the designs. Please refer to the design template again and compare the differences between your design. For example:

- Your page elements are aligned to the left whereas the design template is centered. Please, let's correct it.
- Your page has no margin to left, bottom and top making it look completely different from the design.
- In your design, the `Add New Book` section is longer in width than the main book store. In the design, they have the same width.
- Please, do not use a different color as you have used green for the book category.